src/me.rs

```diff
@@ -12,111 +12,130 @@ use context::BLOCK_TO_PLANE_SHIFT;
 use context::MI_SIZE;
 use partition::*;
 use plane::*;
-use util::*;
 use FrameInvariants;
 use FrameState;
 
-use libc;
-
-#[cfg(all(any(target_arch = "x86", target_arch = "x86_64"), not(windows)))]
-extern {
-  fn rav1e_sad_4x4_hbd_ssse3(
-    src: *const u16, src_stride: libc::ptrdiff_t, dst: *const u16,
-    dst_stride: libc::ptrdiff_t
-  ) -> u32;
-}
+#[cfg(all(target_arch = "x86_64", not(windows), feature = "nasm"))]
+mod nasm {
+  use libc;
+  use plane::*;
+  use util::*;
+
+  extern {
```

Comment:
You could combine the two externs if you are going to remove the separate x86 section.